Monika After Story/game/script-holidays.rpy

```diff
@@ -6235,6 +6256,13 @@ label mas_f14_monika_valentines_intro:
             m 3tsb "Ahaha! I'm just kidding...{w=0.5}do you like my outfit?"
             call mas_f14_sun_dress_outro
 
+        elif not has_shoulderless:
+            m 2eua "What do you think of my outfit?"
+            call mas_f14_intro_blackpink_dress
```

Comment:
Are you sure we don't need to run code to unlock the new dress here as well as save persistent?

Reply:
ig it can't hurt? but this should only be reachable if we've went thru the autoload first

Reply:
We do that for sundress that's why I'm asking.